lib/parameters.py

```diff
@@ -469,6 +469,56 @@ def gen_avc_rqp_parameters(spec, profiles):
   params = gen_avc_rqp_variants(spec, profiles)
   return keys, params
 
+def gen_avc_seek_variants(spec):
+  for case, params in spec.items():
+    variants = params.get("variants", dict()).get("seek", [])
+    for variant in variants:
+        rcmode = variant["rcmode"]
+        bitrate = variant["bitrate"]
+
+        # Update maxrate according to rcmode
+        if "cbr" == rcmode:
+          variant.update(maxrate = bitrate)
+        elif "vbr" == rcmode:
+          variant.update(maxrate = bitrate * 2)
+        else:
+          variant.update(maxrate = None)
+
+        yield [
+            case, rcmode, bitrate, variant.get("maxrate", None),
+            variant.get("fps", 25), variant.get("seek", 1)
+            ]
+
+def gen_avc_seek_parameters(spec):
+  keys = ("case", "rcmode", "bitrate", "maxrate", "fps", "seek")
+  params = gen_avc_seek_variants(spec)
+  return keys, params
+
+def gen_avc_seek_lp_variants(spec):
+  for case, params in spec.items():
+    variants = params.get("variants", dict()).get("seek_lp", [])
+    for variant in variants:
+        rcmode = variant["rcmode"]
+        bitrate = variant["bitrate"]
+
+        # Update maxrate according to rcmode
+        if "cbr" == rcmode:
+          variant.update(maxrate = bitrate)
+        elif "vbr" == rcmode:
+          variant.update(maxrate = bitrate * 2)
+        else:
+          variant.update(maxrate = None)
+
+        yield [
+            case, rcmode, bitrate, variant.get("maxrate", None),
+            variant.get("fps", 25), variant.get("seek", 1)
+            ]
+
+def gen_avc_seek_lp_parameters(spec):
+  keys = ("case", "rcmode", "bitrate", "maxrate", "fps", "seek")
+  params = gen_avc_seek_lp_variants(spec)
+  return keys, params
+
```

Comment:
#679 merged.  Thus, we don't need the duplicate code. Instead, lets do:
```python
gen_avc_seek_parameters = gen_vp9_seek_parameters
gen_avc_seek_lp_parameters = gen_vp9_seek_lp_parameters
```

---
As a side note, in the future it would be nice if we can refactor this file to consolidate all the other duplicate code.  There is a lot of it.  But, it doesn't need to be done for this PR.